packages/api-clients/open-api/bffApi.yml

```
@@ -17781,6 +17781,9 @@ components:
           format: date-time
         consumer:
           $ref: "#/components/schemas/CompactOrganization"
+        adminId:
+          type: string
+          format: uuid
```

```suggestion
          $ref: "#/components/schemas/CompactOrganization"
```
In the BFF we should enhance the id, you will need to modify `enhanceClient` too